libs/fetcher.js

```diff
@@ -335,7 +335,14 @@ Fetcher.middleware = function (options) {
                     }
                     if (err) {
                         var errResponse = getErrorResponse(err);
-                        res.status(errResponse.statusCode).json(responseFormatter(req, res, errResponse.output));
+                        if (req.query && req.query.returnMeta) {
```

Comment:
@Vijar can you explain this part? will this break responseFormatter since output is not passed in directly?